packages/design-system/src/components/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -142,6 +181,69 @@ export default defineComponent({
   },
   setup(props) {
     const { $gettext } = useGettext()
+    const visibleItems = ref<BreadcrumbItem[]>([])
+    const hiddenItems = ref<BreadcrumbItem[]>([])
+    const displayItems = ref<BreadcrumbItem[]>(props.items.slice())
+
+    const getBreadcrumbElement = (id): HTMLElement => {
+      return document.querySelector(`[data-item-id="${id}"]`)
```

Comment:
Changing the key from `data-id` to `data-item-id` was not what I meant... or at least that's not more specific than before. What I had in mind was that you make the document query more specific to the breadcrumbs, e.g. by adding the `oc-breadcrumb-list` class to the selector. That should be `.oc-breadcrumb-list [data-id="${id}"]`. That way the document query only checks for elements inside the breadcrumbs. If another part of the document also has a `data-id` attribute in the future (for different purposes) it will not cause conflicts for your breadcrumb item document query.

Reply:
yes right didn't see it somehow